src/test/resources/Homework18.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="Homework18">
+    <parameter name="BaseURL" value="https://qa.koel.app/"></parameter>
+
+    <test name="Homework18" preserve-order="false">
+        <classes>
+            <class name="Homework18"/>
+                <methods>
+                      <include name="playSong"/>
```

Comment:
You need to include correct function name of class name to test. And in the your `Homewrok18 `class above, I can see you are defining function name test is  `PlaySong` is not `playSong`
Please upate it